### PR TITLE
360 Video Example

### PR DESCRIPTION
**Pull request by chrisgervang**
For #

#### Background
Utilize the new video support in textures to create a 360 video with a SimpleMeshLayer and FirstPersonView.

#### Change List
- adds FirstPersonView 360 video example to "views" section of website.

### CODE REVIEW

**Comment by chrisgervang** [comment]
It'll look black until we fix the video feature. If you scroll out far enough you'll see we're inside a sphere. Clicking plays the video (autoplay doesn't always work). I haven't been able to get this example to show the video at all yet, and there aren't any concerning webgl warnings.

**Comment by chrisgervang** [comment]
An image works, added to the example for reference. The FPV controller doesn't feel exactly right, but I think that's tweak-able after the video works.

File: examples/website/360-video/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This is a minimal standalone version of the SimpleMeshLayer example
```
Comment (Pessimistress) [inline examples/website/360-video/README.md:1]:
```suggestion
This is a minimal standalone version of the FirstPersonView example
```

File: examples/website/360-video/app.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    const source = document.createElement('source');
+    source.src = VIDEO_URL;
+    videoEl.append(source);
+
+    return videoEl;
+  }, []);
+
+  const layer = new SimpleMeshLayer({
+    id: 'simplemesh-layer',
```
Comment (Pessimistress) [inline examples/website/360-video/app.js:54]:
```suggestion
    id: 'video-sphere',
```

File: examples/website/360-video/index.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>deck.gl SimpleMesh Layer Example</title>
```
Comment (Pessimistress) [inline examples/website/360-video/index.html:5]:
```suggestion
  <title>deck.gl 360 Video Example</title>
```

File: examples/website/360-video/package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "simplemesh-layer-demo",
```
Comment (Pessimistress) [inline examples/website/360-video/package.json:2]:
```suggestion
  "name": "360-video",
```

**Review by Pessimistress — APPROVED** [review]

**Pull request merged**